_sass/color_schemes/memo.scss

```diff
@@ -1,3 +1,58 @@
 // Change default font family
 $body-font-family: "Inter", system-ui, -apple-system, blinkmacsystemfont,
   "Segoe UI", roboto, "Helvetica Neue", arial, sans-serif;
+
+  // Typography
+
+  $body-font-family: "Inter";
+  $root-font-size: 15px; // Base font-size for rems
```

Comment:
How are these variables used? Do they override the ones in our theme? 

If so, is it the theme that is responsible for translating pixels into rems?

Reply:
> How are these variables used? Do they override the ones in our theme?

In short, yes. It's part of the configuration options provided by the theme. See [documentation](https://just-the-docs.github.io/just-the-docs/docs/customization/#define-a-custom-scheme) and the [list of variables](https://github.com/just-the-docs/just-the-docs/blob/adbc25dec583e4d19f1511e53dce47785cfd00ec/_sass/support/_variables.scss) available.

> If so, is it the theme that is responsible for translating pixels into rems?

I guess so because the [default variables](https://github.com/just-the-docs/just-the-docs/blob/main/_sass/support/_variables.scss#L6) provided by the theme mix pixels and rems.